### ISSUE TITLE
Feature to display multiple months in Calendar

### ISSUE THREAD

**Issue by lebnic** [primefaces/primeng#4759]
### There is no guarantee in receiving a response in GitHub Issue Tracker, If you'd like to secure our response, you may consider *PrimeNG PRO Support* where support is provided within 4 business hours

**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => Search github for a similar issue or PR before submitting
[x] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

As discussed in this article https://www.nngroup.com/articles/date-input/ published by the Nielsen Norman Group : "_for selecting a date range [it is preferable to] display two months side by side_"
![image](https://user-images.githubusercontent.com/6682795/34571053-8d7a39da-f13b-11e7-818c-a27e298c0ef3.png)

Google Analytics' implementation is efficient too
![image](https://user-images.githubusercontent.com/6682795/34573561-98b45cfc-f142-11e7-9a68-cb48e288178b.png)

**Comment by winnemucca** [comment]
@cagataycivici Our team just moved this library a few days ago.  So this is all pretty new to me.    I see this was added March 26th as a new feature.   I'm not seeing/confusing the property that is for multiple months.

**Comment by cagataycivici** [comment]
It was a new feature with no target milestone until now, we plan to do it in 6.1.0.

**Comment by winnemucca** [comment]
Thank you for the heads up!

**Comment by cagataycivici** [comment]
Available in 6.0.1.

**Comment by winnemucca** [comment]
@cagataycivici Thank you for the hard work!
